barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/ultra_circuit_builder.cpp

```diff
@@ -451,10 +458,175 @@ template <typename FF> void UltraCircuitBuilder_<FF>::create_ecc_add_gate(const
     q_sort.emplace_back(0);
     q_lookup_type.emplace_back(0);
     q_elliptic.emplace_back(0);
+    q_elliptic_double.emplace_back(0);
     q_aux.emplace_back(0);
     ++this->num_gates;
 }
 
+/**
+ * @brief Create an elliptic curve addition gate
+ *
+ * @details x and y are defined over scalar field. Addition can handle applying the curve endomorphism to one of the
+ * points being summed at the time of addition.
```

Comment:
- Is this meant to be for point addition or point doubling?
- minor point: I think we should be specific about the endomorphisms in question; I assume this description was talking about point doubling

Reply:
point doubling. I copied the comments from `ecc_add_gate` and forgot to change. The endomorphism stuff doesn't apply here

Reply:
fixed